platforms/software/testing-base/src/main/java/org/gradle/api/internal/tasks/testing/DecoratingTestDescriptor.java

```diff
@@ -59,9 +59,12 @@ public String getClassName() {
         return descriptor.getClassName();
     }
 
-    @Override
     public String getMethodName() {
```

Comment:
🤔 Class seems to be frozen as an exeption from `@NonNullApi`, consider removing the line and adding the annotations: https://github.com/gradle/gradle/blob/1a5e48701b602def428d9c79e3793adff7a4f798/testing/architecture-test/src/changes/archunit_store/internal-api-nullability.txt#L721